### PR TITLE
BLER valdiation curves

### PR DESCRIPTION
**Pull request by jakthra**
What's the status on this branch? I see there's some errors for 64 QAM, nonetheless I think we should have a script for producing the validation curves. This should be merged imo.

### CODE REVIEW

**Comment by jakthra** [comment]
The BER error curves rougly makes sense. However, there seems to be an issues with the BLER. Not sure what is the root cause of this. I say we merge this script and create an issue on the BLER. 
image

File: examples/PDSCHLTEExample.m [hunk 1]
```diff
@@ -0,0 +1,573 @@
...
+% * TM2: Transmit diversity
+% * TM3: Open loop codebook based precoding: Cyclic Delay Diversity (CDD)
+% * TM4: Closed loop codebook based spatial multiplexing
+% * TM6: Single layer closed loop codebook based spatial multiplexing
+%
+% The example also shows how to parameterize and customize the settings for
+% the different TMs. It also supports the use of 
+% Parallel Computing Toolbox(TM) to reduce effective simulation time.
+
+% Copyright 2016-2018 The MathWorks, Inc.
```
Comment (MonsterMarkus) [inline examples/PDSCHLTEExample.m]: How come we are adding a matlab test script under examples? This uses none of the functions we've build and is available for anyone with matlab access anyway.

Comment (jakthra) [reply examples/PDSCHLTEExample.m]: Where did this script come from? 🤣 Who commited it?

Comment (MonsterMarkus) [reply examples/PDSCHLTEExample.m]: I think I accidentally did when I tried to figure out what was wrong. I'll just remove it.
